spec/unit/hanami/reloader/commands/install_spec.rb

```diff
@@ -33,17 +33,14 @@
       expect(fs.read("Gemfile")).to include(gemfile)
 
       # Guardfile
+      matcher = Hanami::Reloader::Commands::Install::MATCHER.inspect.gsub("/^", "^").gsub("$/i", "$}i").gsub('*\\.', "*.").gsub("\\\\\\", %(\\))
```

Comment:
@timriley Correct. This is cumbersome.

I can't directly do `"#{Hanami::Reloader::Commands::Install::MATCHER.inspect}"` in the template because it will generate a slightly different regular expression.

So I had to:

1. Introduce the constant
2. Write unit tests for it (`spec/unit/hanami/reloader/matcher_spec.rb`)
3. Copy and paste the literal of `MATCHER` into the template
4. Doing this wild string manipulation with `MATCHER` into `spec/unit/hanami/reloader/commands/install_spec.rb`

🙈